pkg/gadget-collection/gadgets/top/block-io/gadget.go

```diff
@@ -48,15 +50,17 @@ func NewFactory() gadgets.TraceFactory {
 }
 
 func (f *TraceFactory) Description() string {
+	cols := columns.MustCreateColumns[types.Stats]()
+
 	t := `biotop shows command generating block I/O, with container details.
 
 The following parameters are supported:
  - %s: Output interval, in seconds. (default %d)
  - %s: Maximum rows to print. (default %d)
  - %s: The field to sort the results by (%s). (default %s)`
-	return fmt.Sprintf(t, types.IntervalParam, types.IntervalDefault,
-		types.MaxRowsParam, types.MaxRowsDefault,
-		types.SortByParam, strings.Join(types.SortBySlice, ","), types.SortByDefault)
+	return fmt.Sprintf(t, top.IntervalParam, top.IntervalDefault,
+		top.MaxRowsParam, top.MaxRowsDefault,
+		top.SortByParam, strings.Join(cols.GetColumnNames(), ","), strings.Join(types.SortByDefault, ","))
```

Comment:
I also think that would be better. Should be the done in kind of the same way as in the `kubectl-gadget` `top` gadgets in this PR

Reply:
I consider it is useful to have this info here because it is used to generate [this doc](https://github.com/inspektor-gadget/inspektor-gadget/blob/main/docs/gadgets/filetop.md) which wouuld be read by people using directly the CRD and not passing through the CLI. Even though we were thinking to unify it with [the guides](https://github.com/inspektor-gadget/inspektor-gadget/tree/main/docs/guides), until then, I would suggest to keep it updated (BTW, it looks like some gadgets are missing 😅).

Reply:
Yes, they are missing :D
The CI doesnt complain when new files are generated by `make manifests generate generate-documentation`.
Intentional or not?